pkg/backup/item_backupper.go

```diff
@@ -147,6 +147,9 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 		return nil
 	}
 
+	if metadata.GetDeletionTimestamp() != nil {
+		return nil
```

Comment:
Let's `log.Info` that we're skipping it because it's being deleted.

Reply:
Done.